webapp/components/Badges.vue

```diff
@@ -10,22 +10,31 @@
       :key="badge.key"
       class="hc-badge-container"
     >
-      <img
+      <hc-image
         :title="badge.key"
-        :src="badge.icon"
+        :image-props="imageProps(badge.icon)"
```

Comment:
```suggestion
        :image-props="{src: badge.icon}"
```
@aonomike that does not work?

Reply:
when I was coding with him @roschaefer this wasn't working because it is calling `this.imageProps` and that was throwing an error, but I haven't pulled it down and played around with it, so I could be off base